src/components/shared/Login.jsx

```diff
@@ -1,54 +1,93 @@
 import { GoogleLogin, GoogleLogout } from "react-google-login";
 import { useState, useEffect } from "react";
+import axios from "axios";
+import LoginForm from "../LoginForm";
 
 const clientId = process.env.REACT_APP_CLIENTID;
 
 const Login = (props) => {
   const [showLoginButton, setShowLoginButton] = useState(true);
   const [showLogoutButton, setShowLogoutButton] = useState(false);
```

Comment:
Do we need two different variable/state for Boolean

Reply:
Removed, as implemented in another way